index.html

```diff
@@ -5186,7 +5182,7 @@ <h3>Content Integrity Protection</h3>
   "type": ["VerifiableCredential", "ExampleAlumniCredential"],
   "credentialSubject": {
     "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    "image": <span class="highlight">"ipfs:/ipfs/QmXfrS3pHerg44zzK6QKQj6JDk8H6cMtQS7pdXbohwNQfK/image"</span>,
+    "image": <span class="highlight">"https://example.com/image"</span>,
```

Comment:
similar to not including direct references to bitcoin or ethereum, IPFS is a product from a specific W3C member... we should avoid the appearance of bias.